dts/bindings/acpi/acpi-device.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for PCIe devices
```

Comment:
"Common fields for ACPI informed based devices"

Reply:
updated